### PR TITLE
Changed the link of contributing guidlines in the PR Template

### LINKED ISSUE #869: [CHORE] - Update PR template to redirect directly to contributing site, rather than CONTRIBUTING file
### What work needs to be performed?

Change the `PULL_REQUEST_TEMPLATE.md` file's contributing guidelines link to redirect directly to the [contributing site](https://docs.beccalyria.com/#/contribute.md) rather than the CONTRIBUTING file, as the file's only contents are primarily to link to the [Developer Certificate of Origin](https://developercertificate.org/) and the contributing site (linked above).

### Additional information

Maybe it's intended to be a 2 step flow, as I'm not sure if the developer certificate of origin is provided in the site anywhere? If its intended, then this issue can be closed as its intended 👍

### PR DESCRIPTION
**Pull request by MukulKolpe**
# Pull Request

## Description

Changed the link of contributing guidelines from CONTRIBUTING.md to the [contribute.md](https://docs.beccalyria.com/#/contribute.md) from the docs in the `PULL_REQUEST_TEMPLATE.md`

## Related Issue

Closes #869 

## Documentation

For _any_ version updates, please verify if the [documentation page](https://www.beccalyria.com/discord-documentation) needs an update. If it does, please [create an issue there](https://github.com/BeccaLyria/discord-documentation/issues/new?assignees=nhcarrigan&labels=%F0%9F%9A%A6+status%3A+awaiting+triage&template=update.md&title=%5BUPDATE%5D) to ensure it is not forgotten.

- [x] My contribution does NOT require a documentation update.
- [ ] My contribution DOES require a documentation update.

### CODE REVIEW

**Comment by nhcarrigan** [comment]
Blocked pending https://github.com/BeccaLyria/discord-documentation/issues/560

**Comment by nhcarrigan** [comment]
Heya @MukulKolpe, 

This looks good and will be approved/merged once unblocked - I've gone ahead and added the `hacktoberfest-accepted` label so you are credited for your contribution here without having to wait. 🙂

**Comment by MukulKolpe** [comment]
Hey, Thank you @nhcarrigan, really appreciate that....Looking forward to keeping on contributing here😀

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Pull Request
 
-<!-- Before contributing, please read our contributing guidelines https://github.com/BeccaLyria/discord-bot/blob/main/CONTRIBUTING.md -->
+<!-- Before contributing, please read our contributing guidelines https://docs.beccalyria.com/#/contribute.md -->
```
Comment (nhcarrigan) [inline .github/PULL_REQUEST_TEMPLATE.md:3]:
```suggestion
<!-- Before contributing, please read our contributing guidelines https://docs.beccalyria.com/#/contribute -->
```

**Review by nhcarrigan — CHANGES_REQUESTED** [review]
Quick tweak.

**Review by nhcarrigan — APPROVED** [review]
Looks good to me!

**Pull request merged**